Sources/SwiftDocCUtilities/ArgumentParsing/Subcommands/Convert.swift

```diff
@@ -138,6 +138,10 @@ extension Docc {
         @Flag(help: "Inherit documentation for inherited symbols")
         public var enableInheritedDocs = false
         
+        
+        @Flag(help: "Treat diagnostics level of warning as error")
```

Comment:
```suggestion
        @Flag(help: "Treat warnings as errors")
```

This is the phrasing the Swift compiler uses https://github.com/apple/swift/blob/main/include/swift/Option/Options.td#L641